pkg/server/server.go

```diff
@@ -390,11 +399,15 @@ func (s *Server) Run(ctx context.Context) error {
 
 		go s.kcpSharedInformerFactory.Apis().V1alpha1().APIExports().Informer().Run(ctx.StopCh)
 		go s.kcpSharedInformerFactory.Apis().V1alpha1().APIBindings().Informer().Run(ctx.StopCh)
+		go s.kcpClusterSharedInformerFactory.Apis().V1alpha1().APIExports().Informer().Run(ctx.StopCh)
+		go s.kcpClusterSharedInformerFactory.Apis().V1alpha1().APIBindings().Informer().Run(ctx.StopCh)
```

Comment:
are these temporary to move incrementally over?

Reply:
yes, if we just replace the kcpSharedInformerFactory with the new ones we'd need to update every controller at once, this way we can do it as a series of smaller PRs and remove the old stuff once nothing is using it anymore.

Reply:
what about kube informers? Are the newly generated ones compatible enough to inject them deep into kube?

Reply:
after this PR they will all generate https://github.com/kcp-dev/code-generator/pull/38